packages/card/src/spectrum-card.css

```diff
@@ -11,231 +11,238 @@ governing permissions and limitations under the License.
+        --spectrum-card-background-color-hover-quiet,
+        var(--spectrum-gray-300)
     );
-    --spectrum-card-body-content-margin-top: var(
-        --spectrum-card-m-body-content-margin-top,
-        var(--spectrum-global-dimension-size-75)
+}
+:host {
+    background-color: var(
+        --highcontrast-card-background-color,
+        var(
+            --mod-spectrum-card-background-color,
+            var(--spectrum-card-background-color)
+        )
     );
-    --spectrum-card-body-padding-top: var(
-        --spectrum-card-m-body-padding-top,
-        var(--spectrum-global-dimension-size-250)
+    border: var(--mod-card-border-width, var(--spectrum-card-border-width))
+        solid transparent;
+    border-color: var(
+        --highcontrast-card-border-color,
+        var(--mod-card-border-color, var(--spectrum-card-border-color))
     );
-    --spectrum-card-body-padding-bottom: var(
-        --spectrum-card-m-body-padding-bottom,
-        var(--spectrum-global-dimension-size-250)
+    border-radius: var(
+        --mod-card-corner-radius,
+        var(--spectrum-card-corner-radius)
     );
-    --spectrum-card-body-padding-left: var(
-        --spectrum-card-m-body-padding-left,
-        var(--spectrum-global-dimension-size-300)
+    box-sizing: border-box;
+    display: inline-flex;
+    flex-direction: column;
+    min-inline-size: var(
+        --mod-card-minimum-width,
+        var(--spectrum-card-minimum-width)
     );
-    --spectrum-card-body-padding-right: var(
-        --spectrum-card-m-body-padding-right,
-        var(--spectrum-global-dimension-size-300)
+    position: relative;
+    -webkit-text-decoration: none;
+    text-decoration: none;
+}
+:host:before {
+    block-size: 100%;
+    content: '';
+    inline-size: 100%;
+    inset-block-start: 0;
+    inset-inline-start: 0;
+    position: absolute;
+}
+:host:after {
+    block-size: 100%;
+    border: 0 solid #0000;
+    border-radius: var(
+        --mod-card-corner-radius,
+        var(--spectrum-card-corner-radius)
     );
-    --spectrum-card-coverphoto-height: var(
-        --spectrum-card-m-coverphoto-height,
-        var(--spectrum-global-dimension-size-1700)
+    content: '';
+    inline-size: 100%;
+    inset-block-start: 0;
+    inset-inline: 0;
+    margin-block-start: calc(
+        var(
+                --mod-card-focus-indicator-width,
+                var(--spectrum-card-focus-indicator-width)
+            ) * -1
     );
-    --spectrum-card-coverphoto-border-bottom-size: var(
-        --spectrum-card-m-coverphoto-border-bottom-size,
-        var(--spectrum-alias-border-size-thin)
+    margin-inline-start: calc(
+        var(
+                --mod-card-focus-indicator-width,
+                var(--spectrum-card-focus-indicator-width)
+            ) * -1
     );
-    --spectrum-card-checkbox-margin: var(
-        --spectrum-card-m-checkbox-margin,
-        var(--spectrum-global-dimension-size-200)
+    position: absolute;
+}
```

Comment:
@mlogsdon18 sorry I missed this last one, but these pseudo elements need to have `pointer-events: none;` or they block interaction with other parts of the page. Do you want to work out a new version of this package now, or should we add this as an override locally in anticipation for that change coming along later?

Reply:
For reference, this is what was causing [these](https://app.circleci.com/pipelines/github/adobe/spectrum-web-components/16382/workflows/83434957-27fd-4a1a-b69d-f354113b673d/jobs/350359/tests) sorts of unit test failures.

Reply:
I fixed this locally, but we'll need this updated in the CSS source, sooner or later. It actually aligns with [this issue](https://github.com/adobe/spectrum-web-components/issues/3357) in Dropzone.